examples/material_gallery/lib/demo/menu_demo.dart

```diff
@@ -94,7 +94,7 @@ class MenuDemoState extends State<MenuDemo> {
                   child: new Text('Context menu item one')
                 ),
                 new PopupMenuItem(
-                  disabled: true,
+                  isDisabled: true,
```

Comment:
"isThreeLine" is probably better than "threeLine" since the latter isn't an adjective. Generally I think my intuition is that arguments that are booleans should be adjective, and arguments that take other types should be nouns. (length:3 -> "the length is 3" vs dens:true -> "this is dense")